src/NuGet.Core/NuGet.Build.Tasks.Pack/NuGet.Build.Tasks.Pack.targets

```diff
@@ -192,8 +198,7 @@ Copyright (c) .NET Foundation. All rights reserved.
     </PropertyGroup>
   </Target>
 
-  <Target Name="GenerateNuspec" DependsOnTargets="$(GenerateNuspecDependsOn);_CalculateInputsOutputsForPack;_GetProjectReferenceVersions" Condition="$(IsPackable) == 'true'"
-          Inputs="@(NuGetPackInput)" Outputs="@(NuGetPackOutput)">
+  <Target Name="GenerateNuspecInternal" DependsOnTargets="$(GenerateNuspecDependsOn);_CalculateInputsOutputsForPack;_GetProjectReferenceVersions">
```

Comment:
we can't change the name of GenerateNuspec target as people can rely on it externally. (we only typically change target names which start with _ )

Reply:
This target has different conditions/Input/Output attributes, does it make sense for *this* target to be the replacement for `GenerateNuspec`, or should we find (or create) a better target to name `GenerateNuspec` that better matches the old one?